dockerfile_functions.sh

```diff
@@ -716,57 +716,57 @@ EOI
 # Setting shell to bash to execute a bash script
 SHELL ["/bin/bash", "-c"]
 
-RUN set -euo pipefail \
-    && unset OPENJ9_JAVA_OPTIONS \
-    && SCC_SIZE="50m" \
-    && SCC_GEN_RUNS_COUNT=3 \
-    && DOWNLOAD_PATH_TOMCAT=/tmp/tomcat \
-    && INSTALL_PATH_TOMCAT=/opt/tomcat-home \
-    && TOMCAT_CHECKSUM="0db27185d9fc3174f2c670f814df3dda8a008b89d1a38a5d96cbbe119767ebfb1cf0bce956b27954aee9be19c4a7b91f2579d967932207976322033a86075f98" \
-    && TOMCAT_DWNLD_URL="https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.35/bin/apache-tomcat-9.0.35.tar.gz" \
+RUN set -euo pipefail; \
```

Comment:
The current script is not optimal in the case of Alpine. For Alpine, we are now introducing a separate layer where we are adding dependencies, which means that they are retained in the final image even if they are deleted in a subsequent layer. Adding the extra dependencies and deleting them has to happen in a single RUN command.

Reply:
I'm a bit confused in the way how to proceed as we are using shell to be `/bin/bash` before the RUN command and if RUN command has the bash installation can we switch from `sh` to `bash` on the fly in the same RUN command ?

Reply:
Let us use the following code, seems to work in ash, sh and dash
```
# SCC_GEN_RUNS_COUNT=10
# for i in $(seq 0 $SCC_GEN_RUNS_COUNT)
> do
>    echo $i
> done
0
1
2
3
4
5
6
7
8
9
10
# SCC_SIZE="50m"
# SCC_SIZE=$(echo $SCC_SIZE | sed 's/.$//')
# SCC_SIZE=$((SCC_SIZE * FULL / 100))
# echo $SCC_SIZE
0
# FULL=80
# SCC_SIZE="50m"
# SCC_SIZE=$(echo $SCC_SIZE | sed 's/.$//')
# SCC_SIZE=$((SCC_SIZE * FULL / 100))
# echo $SCC_SIZE
```